app-libs/form-component/src/layout-components/Number/Number.module.css

```
@@ -0,0 +1,37 @@
+  gap: var(--ds-size-1);
+  align-items: flex-start;
+}
+
+.labelRow {
+  display: flex;
+  align-items: center;
+  gap: var(--ds-size-2);
+}
+
+.labelContent {
+  word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)